Dockerfile.updater-core

```diff
@@ -42,9 +42,6 @@ ARG RUBY_INSTALL_VERSION=0.8.5
 
 # When bumping Bundler, need to also regenerate `updater/Gemfile.lock` via `bundle update --bundler`
 ARG BUNDLER_V2_VERSION=2.3.25
-ENV BUNDLE_SILENCE_ROOT_WARNING=1
```

Comment:
Does this need to be added in a later stage or is that already happening? (On my phone so can't check source)

Reply:
My rationale was since no longer setting it in the updater image has not broken anything, it was not needed. This makes sense since I don't think we run Bundler as root. Maybe we did this in the past and that's why we needed this to tell Bundler: "We know this is bad, but we know what we're doing". But I think we're using better practices now.